taipy/core/data/data_node.py

```diff
@@ -567,6 +572,19 @@ def get_last_edit(self) -> Optional[Edit]:
         """
         return self._edits[-1] if self._edits else None
 
+    def get_rank(self) -> int:
```

Comment:
Maybe add a docstring here explain the returned rank since this is a public method

Reply:
Will do it, tx